spec/support/authentication.rb

```diff
@@ -0,0 +1,117 @@
+module AuthenticationHelper
+  def sign_in_as(user)
+    mock_auth_for(user)
+    visit root_path
+    click_link "Sign In With Github"
+  end
+
+  def mock_auth_for(user)
+    OmniAuth.config.mock_auth[:github] = OmniAuth::AuthHash.new({
+      "provider" => user.provider,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.